bitbots_hcm/src/bitbots_hcm/hcm_dsd/hcm_blackboard.py

```diff
@@ -114,9 +121,37 @@ def __init__(self):
         self.hacky_sequence_dynup_running = False
 
         # kicking
-        self.last_kick_feedback = None      # type: rospy.Time
+        self.last_kick_feedback = None  # type: rospy.Time
+
+        # direct messages for falling classier
+        # todo needs refactoring
+
+        rospack = rospkg.RosPack()
+        rospack.list()
+        path = rospack.get_path('bitbots_hcm')
+        smooth_threshold = rospy.get_param("/smooth_threshold", 10)
```

Comment:
The parameter name should be something like `hcm/smooth_threshold` and should be added to the config file.

Reply:
Did you forget to push something or why did you resolve this?

Reply:
should be fixed now